hc/accounts/tests/test_profile.py

```diff
@@ -108,3 +120,8 @@ def test_it_shows_badges(self):
         self.assertNotContains(r, "bobs-tag.svg")
 
     ### Test it creates and revokes API key
+    def test_it_creates_and_revokes_api_key(self):
```

Comment:
We should be testing out this https://github.com/andela/healthchecks_akira/blob/df137bfdad57cb53bcf31bdc92056cb4f316f921/hc/accounts/views.py#L150